### ISSUE TITLE
Makefile to build local charts fails on Ubuntu 20.04LTS

### ISSUE THREAD

**Issue by matthewmodestino** [splunk/splunk-connect-for-kubernetes#432]
**What happened**:

Tried to pull main branch and run `make build`

**What you expected to happen**:

to build local versions of helm chart in `build` dir. 

**How to reproduce it (as minimally and precisely as possible)**:

Spin up Ubuntu 20.04LTS (not minimal)

make sure helm is installed
```
sudo snap install helm --channel=3.2/stable --classic

```
run `make build`

```
make build
#!/usr/bin/env bash
please install yq first
make: *** [Makefile:27: version-update] Error 1
```
New requirement as of 1.4.2, i assume. 

```
sudo snap install yq
```
Fwiw, i tried apt install too...not sure where the default dir was but it was failing to detect install. 
```
which yq
/snap/bin/yq
```
Updated `./tools/update_charts_version.sh` to point to my snap install location

```
if [ -f "/snap/bin/yq" ]; then
    echo "yq already installed"    
else 
    echo "please install yq first"
    exit 1
fi

```

tried `make build`

```
yq already installed
VERSION 1.4.2
Updating Charts' version
rm: cannot remove 'build/*': No such file or directory
make: *** [Makefile:6: create-dir] Error 1
```

Updated `Makefile`

```
create-dir: version-update
        @mkdir -p build
        @rm -rf build/*
```

```
make build
yq already installed
VERSION 1.4.2
Updating Charts' version
Successfully packaged chart and saved it to: build/splunk-connect-for-kubernetes-1.4.2.tgz
Successfully packaged chart and saved it to: build/splunk-kubernetes-logging-1.4.2.tgz
Successfully packaged chart and saved it to: build/splunk-kubernetes-objects-1.4.2.tgz
Successfully packaged chart and saved it to: build/splunk-kubernetes-metrics-1.4.2.tgz
```

Shout out to Amir and Aalok for bumping their heads!

**Anything else we need to know?**:

**Environment**:

Ubuntu GCP VM with main branch pulled from repo

- Kubernetes version (use `kubectl version`):
- Ruby version (use `ruby --version`):
- OS (e.g: `cat /etc/os-release`):
- Splunk version:
- Others:

**Comment by rockb1017** [comment]
will include this to patch release

**Comment by matthewmodestino** [comment]
what do you have in mind? is there's a more global way, or if we just have to document pre-requisites for popular distros?

**Comment by rockb1017** [comment]
actually, just to build charts, we don't need to run `yq` stuff. I will skip that step. :)

**Comment by rockb1017** [comment]
Fix is being released with 1.4.3
Thank you!
